app/Plugin/CoreApi/Config/routes.php

```diff
@@ -71,16 +71,10 @@
   )
 );
 
-// Scoped identifiers are not parsed properly because they are perceived as file extensions
-// Enable extensions parse to resolve this problem
-Router::parseExtensions('json', 'xml');
 
-
-/*
-
-Router::connect(
 // XXX This needs to trigger identifier assignment and maybe some other stuff
 //     provisioning should only fire after all models are saved
```

Comment:
It looks like you did this, so we can probably delete this comment?

Reply:
Done